shared/css/conversation.css

```diff
@@ -364,11 +364,59 @@ html[dir="rtl"] .settings-menu.dropdown-menu {
 /*
  * For any audio-only streams, we want to display our own background
  */
+.avatar-container {
+  width:  100%;
+  height: 100%;
+  background-color: #d8d8d8;
+  overflow: hidden;
+}
+
+.media-wrapper > .remote > .avatar-container {
```

Comment:
General comment: the component you're dealing with here is the `AvatarView`, which in the code is set up as class `avatar-container`.

Therefore all css pertaining to the display of items within `avatar-container`, should only be referenced at the `avatar-container` level. The only things that should reference child selectors from other classes outside of it should be positional/layout items.

If, for example, you want to specify a different background colour for a remote element versus a local one, then you should expose a new class name on the `avatar-container`, e.g.

```css
.avatar-container.remote {
  background-colour: #00a9dc
}
```

This way, the component can be re-used or moved to a different place in the hierarchy without needing to redo lots of css.